app/src/pages/api/cron/selfDestruct.ts

```diff
@@ -0,0 +1,45 @@
+import { NextApiRequest, NextApiResponse } from "next";
```

Comment:
Let's make the endpoint file to be: `self-destruct.ts` to keep naming conventions ok.